src/main/java/io/lettuce/core/SetArgs.java

```diff
@@ -39,6 +40,8 @@
 
     private boolean keepttl = false;
 
+    private boolean get = false;
```

Comment:
We do not need this option here because:
1) `set` would not be able to decode value in case of `true` value
2) `setGet` would not be able to decode value in case of `false` value either

This param should not be exposed and manually set inside CommandBuilder